src/instructlab/config/init.py

```diff
@@ -117,53 +121,74 @@ def init(
-        )
-        click.echo("[0] No profile (CPU, Apple Metal, AMD ROCm)")
-        for i, value in enumerate(profile_options):
-            click.echo(f"[{i+1}] {value}")
-        train_profile_selection = click.prompt(
-            "Enter the number of your choice [hit enter for no profile]",
-            type=int,
-            default=0,
-        )
-        if 1 <= train_profile_selection <= len(profile_options):
-            click.echo(f"You selected: {profile_options[train_profile_selection - 1]}")
-            cfg.train = read_train_profile(
-                join(
-                    DEFAULTS.TRAIN_PROFILE_DIR,
-                    train_profile_filenames[train_profile_selection - 1],
-                )
+        (
+            chosen_profile,
+            chosen_vram,
+            detected_train_profile,
+            edited_cfg,
+        ) = hw_auto_detect()
+        yn = None
+        if chosen_vram is not None and chosen_profile is not None:
+            click.echo(
+                f"We chose {chosen_profile} as your designated training profile. This is for systems with {chosen_vram} GB of vRAM."
             )
-        elif train_profile_selection == 0:
+            if edited_cfg:
+                click.echo(
+                    "This profile is the best approximation for your system based off of the amount of vRAM. We modified it to match the number of GPUs you have."
+                )
+            yn = click.confirm("Is this profile correct?", default=True)
+        if yn is None or not yn:
+            train_profile_filenames = sorted(listdir(DEFAULTS.TRAIN_PROFILE_DIR))
+            # generate human-friendly text out of train profile filenames
+            profile_options = []
+            for filename in train_profile_filenames:
+                # one gpu type
+                split_filename = filename.split("_")
+                if len(split_filename) == 2:
+                    gputype1 = split_filename[0]
+                    num_gpu = split_filename[1].split(".")[0]
+                    profile_options.append(f"Nvidia {gputype1} {num_gpu} ({filename})")
+                # two gpu types
+                elif len(split_filename) == 3:
+                    gputype1 = split_filename[0]
+                    gputype2 = split_filename[1]
+                    num_gpu = split_filename[2].split(".")[0]
+                    profile_options.append(
+                        f"Nvidia {gputype1}/{gputype2} {num_gpu} ({filename})"
+                    )
             click.echo(
-                "No profile selected - any hardware acceleration for training must be configured manually."
+                "Please choose a train profile to use.\nTrain profiles assist with the complexity of configuring specific GPU hardware with the InstructLab Training library.\nYou can still take advantage of hardware acceleration for training even if your hardware is not listed."
```

Comment:
Nit: ... complexity of configuring InstructLab training for specific GPU hardware.